keps/sig-apps/2232-jobs-stopped/README.md

```diff
@@ -234,9 +250,9 @@ type JobSpec struct {
 	// a Job is created in the stopped state, no Pods are created by the job
 	// controller. If a Job enters the stopped state (i.e. the flag goes from
 	// false to true), the job controller will delete all Pods associated with
-	// this job. This field is generally used by higher-level orchestrators that
-	// need more control over when the Pods of a Job are created.
-	Stopped *bool
+        // this job. This field is generally used by higher-level controllers that
```

Comment:
nit: format looks off.

Specify the default value: false

But, I think this line shouldn't be part of the API documentation.